src/contracts/krAsset/IKreskoAssetIssuer.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.14;
+
```

Comment:
Add a comment here describing why `IKreskoAssetIssuer` exists in addition to `IKreskoAssetAnchor`, something similar to the justification in the PR description.